app/stac_api/models.py

```diff
@@ -510,7 +510,16 @@ def upload_asset_to_path_hook(instance, filename=None):
     Returns:
         Asset file path to use on S3
     '''
-    logger.debug('Start computing asset file %s multihash', filename)
+    logger.debug(
+        'Start computing asset file %s multihash (file size: %s MB)',
+        filename,
+        int(instance.file.size / 1024**2),
```

Comment:
Based on comment above remove the type casting to `int`

Reply:
:+1: